index.js

```diff
@@ -0,0 +1,236 @@
+const RELEVANT_FIELDS =
+  "tld,name,capital,flags,population,region,subregion,currencies,languages,borders";
+
+const getRelevantURL = (url) => `${url}?fields=${RELEVANT_FIELDS}`;
+
+const freeChildren = (
+  container = document.getElementById("countries-container")
+) => {
+  while (container.firstChild) {
+    container.removeChild(container.firstChild);
+  }
+};
+
+const addInfoLi = (infoType, infoValue) => {
```

Comment:
What if you will use a different html element (and not Li)? Will you change the function's name? and then replace all the usages?
The name should suggest about the implementation.
Consider renaming to: `addInfoElement`